src/ckb-daemon/usb_mac.c

```diff
@@ -271,6 +271,67 @@ static void pipecomplete(void* refcon, IOReturn result, void* arg0){
     (*handle)->ReadPipeAsync(handle, ctx->pipe, buffer, ctx->maxsize, pipecomplete, ctx);
 }
 
+CFMachPortRef event_tap;
+
+CGEventRef cg_callback(CGEventTapProxy proxy, CGEventType type, CGEventRef event, void* refcon) {
```

Comment:
What does this function do exactly? Why is it called cg_callback? Can't there be no other CoreGraphics-related callbacks inside this function?